pkg/test/resourcefixture/testdata/basic/spanner/v1beta1/spannerdatabase/_generated_object_spannerdatabase.golden.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: spanner.cnrm.cloud.google.com/v1beta1
+kind: SpannerDatabase
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/mutable-but-unreadable-fields: '{"spec":{"ddl":["CREATE
+      TABLE t1 (t1 INT64 NOT NULL,) PRIMARY KEY(t1)"]}}'
+    cnrm.cloud.google.com/project-id: ${projectId}
+    cnrm.cloud.google.com/state-into-spec: merge
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: spannerdatabase-test
```

Comment:
nit: There's some [naming convention requirement](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#naming-conventions-special-variables-and-constants-in-testdata). I think this should be spannerdatabase-${uniqueId}

Reply:
Fixed, thanks :) I think I had accidentally changed this for manual testing.